### PR TITLE
Task/replace graph

### PR DESCRIPTION
**Pull request by sophistifunk**
# Description

Updates BO to use the newest graph control and namespaced styles (from ux-widgets).

### CODE REVIEW

**Comment by sophistifunk** [comment]
Not quite ready for merge yet, just want a PR open for running ATH.

File: blueocean-dashboard/package.json [hunk 1]
```diff
@@ -60,8 +60,9 @@
   "dependencies": {
     "@jenkins-cd/blueocean-core-js": "0.0.171",
     "@jenkins-cd/design-language": "0.0.162",
-    "@jenkins-cd/js-modules": "0.0.10",
     "@jenkins-cd/js-extensions": "0.0.44",
+    "@jenkins-cd/js-modules": "0.0.10",
+    "@jenkins-cd/pipeline-graph-widget": "^0.1.2",
```
Comment (NicuPascu) [inline blueocean-dashboard/package.json]: caret :D

**Comment by sophistifunk** [comment]
NFI why there's no option for "this is done" rather than "dismiss" but ¯\_(ツ)_/¯

**Comment by NicuPascu** [comment]
LGTM :bee:

**Comment by sophistifunk** [comment]
Hmm. One of these ATH failures looks like it might be legit...

**Comment by sophistifunk** [comment]
`RestartStageTest` is a valid failure, still trying to lock down the exact changes to bring it back to life. Don't think the other failures are though.

**Pull request merged**